src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -205,16 +205,19 @@ public Subject() {
     public Subject(boolean readOnly, Set<? extends Principal> principals,
                    Set<?> pubCredentials, Set<?> privCredentials)
     {
-        collectionNullClean(principals);
-        collectionNullClean(pubCredentials);
-        collectionNullClean(privCredentials);
+        LinkedList<Principal> princList
+                = collectionNullClean(principals);
+        LinkedList<Object> pubCredsList
+                = collectionNullClean(pubCredentials);
+        LinkedList<Object> privCredsList
+                = collectionNullClean(privCredentials);
 
         this.principals = Collections.synchronizedSet(new SecureSet<>
-                                (this, PRINCIPAL_SET, principals));
+                                (this, PRINCIPAL_SET, princList));
         this.pubCredentials = Collections.synchronizedSet(new SecureSet<>
-                                (this, PUB_CREDENTIAL_SET, pubCredentials));
+                                (this, PUB_CREDENTIAL_SET, pubCredsList));
         this.privCredentials = Collections.synchronizedSet(new SecureSet<>
-                                (this, PRIV_CREDENTIAL_SET, privCredentials));
+                                (this, PRIV_CREDENTIAL_SET, privCredsList));
         this.readOnly = readOnly;
```

Comment:
Indentation (in existing code too) is unusual

Reply:
I'll move `new SecureSet<>` to the next line and indent 8 spaces.